nengo_loihi/loihi_cx.py

```diff
@@ -101,17 +102,25 @@ def add_probe(self, probe):
         assert probe.target is self
         self.probes.append(probe)
 
-    def set_decay_U(self, tau_s, dt):
-        self.decayU[:] = 1 if tau_s == 0 else -np.expm1(-dt/np.asarray(tau_s))
+    def set_decay_U(self, tau_s, dt, default=False, overwrite=False):
+        if default and self.decayU_set:
+            return
 
-    def configure_filter(self, tau_s, dt=0.001):
-        """Synaptic input filter for Cx."""
+        decayU = 1 if tau_s == 0 else -np.expm1(-dt/np.asarray(tau_s))
+        if not overwrite and self.decayU_set and not np.allclose(
+                decayU, self.decayU):
+            raise ValueError(
```

Comment:
I would make this a `BuildError` or `ValidationError` (not sure when this error might be raised, but I suspect it's during the build).

Reply:
Will do